core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -308,6 +308,15 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
   private static final String SHIFTLEFT = "<<";
   private static final String BITNOT = "~";
 
+  public static final Map<String, String> BQ_VALID_ESCAPE_SEQUENCES =
```

Comment:
Rename to 'STRING_LITERAL_ESCAPE_SEQUENCES'

Reply:
Done